Dev/Kaiten/kaiten-dashboard.5m.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash
+# Metadata allows your plugin to show up in the app, and website.
+#
+#  <xbar.title>Kaiten Dashboard</xbar.title>
+#  <xbar.version>v1.0</xbar.version>
+#  <xbar.author>Riccardo Ancona</xbar.author>
+#  <xbar.author.github>raasoft</xbar.author.github>
+#  <xbar.desc>This plugin displays the cards in your Kaiten dashboard</xbar.desc>
+#  <xbar.image>https://i.gyazo.com/ad9cfe01c760a5887e9be8b298479aac.png</xbar.image>
```

Comment:
This image will look a little too small in the UI. Perhaps you can crop it to focus more on the actual menu?